ambry-api/src/main/java/com.github.ambry/config/PerformanceConfig.java

```diff
@@ -0,0 +1,138 @@
+/**
+ * Copyright 2019 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.github.ambry.config;
+
+/**
+ * Configuration parameters required to help evaluate performance of Ambry, i.e latency, health.
+ */
+public class PerformanceConfig {
+
+  /**
+   * The time to first byte (millisecond) threshold for success GET request(2xx status code). The request is considered
+   * "Unsatisfied" if the time exceeds this threshold.
+   */
+  @Config("performance.success.get.time.to.first.byte.threshold")
```

Comment:
Take a look at `FrontendConfig::restRequestQuota`. For that config, Ze used a json object so that we didn't have to make a separate config for each rest method.

Reply:
good point, I have removed unnecessary configs.